client/app/components/land-use-action.hbs

```diff
@@ -5,9 +5,9 @@
       supportsDataTestProperties={{true}}
       @placeholder="-- select an action --"
       @searchEnabled={{false}}
-      @options={{this.landUseActionOptions}}
+      @options={{this.sortedAvailableLandUseActionOptions}}
       @onchange={{fn (mut this.selectedLandUseAction)}}
-      @onclose={{fn this.setSelectedActionsArray}}
+      @onclose={{fn this.addNewLandUseActionSelection}}
```

Comment:
Flag for later: if we close the dropdown without selecting anything, we generate a blank Land Use Action input box (not tied to any PAS form attributes)

Reply:
Good catch! Can you make the action check for this? `if this.selectedAction //...`

Reply:
Good idea, done!